src/numba/openmp/__init__.py

```diff
@@ -7404,6 +7398,7 @@ def NUMBER(self, args):
                        | data_default_clause
                        | data_sharing_clause
                 //     | reduction_default_only_clause
+                       | reduction_clause
```

Comment:
This the only one missing reduction_clause?  How is reduction_default_only_clause different than reduction clause?  I think I probably added the complete list and commented out stuff not supported so perhaps this should be reduction_default_only_clause instead?

Reply:
I don't see the definition of `reduction_default_only_clause` in the grammar, but I see the definition of `reduction_clause`. I suppose I should just delete the commented out line?

Reply:
I briefly looked for the grammar I was copying from but I think we need to find that grammar again and see how reduction_clause is different than reduction_default_only_clause.  I didn't copy in every single clause like this...was only doing as necessary.